src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import CommentCard from './CommentCard';
+import { ITEM } from '../../constants/url';
+
+import './Comment.scss';
+import '../Common/Loader/loader.scss';
+
+export class Comment extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true,
+      content: {},
+      error: false
+    }
+
+    //Abort Controller is an experimental feature and doesnt work on IE.
+    //Use Axios for better handling. I am using this to be familiar with the Api.
+    this.controller = new AbortController();
```

Comment:
Great.

But, don't do heavy work on constructor instead create separate function and call on componentDidMount.